src/proc/colorizer.h

```diff
@@ -93,19 +95,26 @@ namespace librealsense {
         size_t _size; float3* _data;
     };
 
-    class colorizer : public stream_filter_processing_block
+    class EXTENSION_API colorizer : public stream_filter_processing_block
     {
     public:
         colorizer();
 
+        static void update_histogram(int* hist, const uint16_t* depth_data, int w, int h);
```

Comment:
Can you remove the static tag? It should also work with multiple colorizer instances scenario

Reply:
I need it static. Please note that the function is static - not the data.
This function works on data provided via arguments, not some static array (as it used to be)
Supporting multiple colorizers is addressed elsewhere in the code